src/modules/gimbal/output_rc.cpp

```diff
@@ -62,25 +62,25 @@ void OutputRC::update(const ControlData &control_data, bool new_setpoints)
 
 	_stream_device_attitude_status();
 
-	// _angle_outputs are in radians, actuator_controls are in [-1, 1]
-	actuator_controls_s actuator_controls{};
-	actuator_controls.control[0] = constrain(
-					       (_angle_outputs[0] + math::radians(_parameters.mnt_off_roll)) *
-					       (1.0f / (math::radians(_parameters.mnt_range_roll / 2.0f))),
-					       -1.f, 1.f);
-	actuator_controls.control[1] = constrain(
-					       (_angle_outputs[1] + math::radians(_parameters.mnt_off_pitch)) *
-					       (1.0f / (math::radians(_parameters.mnt_range_pitch / 2.0f))),
-					       -1.f, 1.f);
-	actuator_controls.control[2] = constrain(
-					       (_angle_outputs[2] + math::radians(_parameters.mnt_off_yaw)) *
-					       (1.0f / (math::radians(_parameters.mnt_range_yaw / 2.0f))),
-					       -1.f, 1.f);
-	actuator_controls.control[3] = constrain(
-					       _retract_gimbal ? _parameters.mnt_ob_lock_mode : _parameters.mnt_ob_norm_mode,
-					       -1.f, 1.f);
-	actuator_controls.timestamp = hrt_absolute_time();
-	_actuator_controls_pub.publish(actuator_controls);
+	// _angle_outputs are in radians, gimbal_controls are in [-1, 1]
+	gimbal_controls_s gimbal_controls{};
+	gimbal_controls.control[gimbal_controls_s::INDEX_ROLL] = constrain(
+				(_angle_outputs[0] + math::radians(_parameters.mnt_off_roll)) *
+				(1.0f / (math::radians(_parameters.mnt_range_roll / 2.0f))),
+				-1.f, 1.f);
+	gimbal_controls.control[gimbal_controls_s::INDEX_ROLL] = constrain(
+				(_angle_outputs[1] + math::radians(_parameters.mnt_off_pitch)) *
+				(1.0f / (math::radians(_parameters.mnt_range_pitch / 2.0f))),
+				-1.f, 1.f);
+	gimbal_controls.control[gimbal_controls_s::INDEX_ROLL] = constrain(
+				(_angle_outputs[2] + math::radians(_parameters.mnt_off_yaw)) *
+				(1.0f / (math::radians(_parameters.mnt_range_yaw / 2.0f))),
+				-1.f, 1.f);
+	gimbal_controls.retract_gimbal = constrain(
```

Comment:
`retract_gimbal` is not used by CA - should we remove it or add to the allocation?

Reply:
After further discussion removed the commit about removing the Gimbal retraction again. We maybe need to have more automation there than I first thought. But this should be in a separate PR, to not derail the focus here on removing the actuator_controls interface. The gimbal retraction mechanism stays broken till then.